.eslintrc

```diff
@@ -26,6 +26,8 @@
     "curly": ["error", "all"],
     "brace-style": ["error"],
 
+    "@typescript-eslint/no-unsafe-declaration-merging": "off",
```

Comment:
Do you know which line required this eslint rule change? Might be better to do an inline disable instead